community/security/src/test/java/org/neo4j/commandline/admin/security/UsersCommandIT.java

```diff
@@ -40,317 +40,8 @@ public void setup()
     public void shouldGetUsageErrorsWithNoSubCommand() throws Throwable
     {
         // When running 'users' with no subcommand, expect usage errors
-        assertFailedUserCommand( "", new String[0], "Missing arguments: expected at least one sub-command as argument",
+        assertFailedSubCommand( "users", "", new String[0], "Missing arguments: expected sub-command argument 'set-password'",
                 "neo4j-admin users <subcommand> [<username>] [<password>]",
-                "Runs several possible sub-commands for managing the native users" );
-    }
-
-    //
-    // Tests for list command
-    //
-
-    @Test
-    public void shouldListDefaultUser() throws Throwable
-    {
-        // Given only default user
-
-        // When running 'list', expect default user
-        assertSuccessfulUserCommand( "list", args(), "neo4j" );
-    }
-
-    @Test
-    public void shouldListNewUser() throws Throwable
-    {
-        // Given a new user
-        createTestUser( "another", "neo4j" );
-
-        // When running 'list', expect only new user (default not created if a new user is created first)
-        assertSuccessfulUserCommand( "list", args(), "another" );
-    }
-
-    @Test
-    public void shouldListSpecifiedUser() throws Throwable
-    {
-        // Given a new user
-        createTestUser( "another", "neo4j" );
-
-        // When running 'list' with filter, expect specified user
-        assertSuccessfulUserCommand( "list", args( "other" ), "another" );
-    }
-
-    //
-    // Tests for set-password command
-    //
-
-    @Test
-    public void shouldGetUsageErrorsWithSetPasswordCommandAndNoArgs() throws Throwable
```

Comment:
Why delete all set-password integration tests? I think these tests help clarify the expected behavior of this command.